ironmon_tracker/MoveData.lua

```diff
@@ -1,3756 +1,3768 @@
 		iscontact = true,
 		summary = "This move is 1% more accurate for each level above the target. Fails if target is higher level.",
 	},
 	{
 		id = "13",
 		name = "Razor Wind",
-		type = PokemonTypes.NORMAL,
+		type = PokemonData.Types.NORMAL,
 		power = "80",
 		pp = "10",
 		accuracy = "100",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		summary = "Does nothing on the turn it is selected. Deals damage on the following turn.",
 	},
 	{
 		id = "14",
 		name = "Swords Dance",
-		type = PokemonTypes.NORMAL,
-		power = NOPOWER,
+		type = PokemonData.Types.NORMAL,
+		power = Constants.NO_POWER,
 		pp = "30",
-		accuracy = PLACEHOLDER,
-		category = MoveCategories.STATUS,
+		accuracy = Constants.BLANKLINE,
+		category = MoveData.Categories.STATUS,
 		summary = "Increases the user's Attack stat by two stages.",
 	},
 	{
 		id = "15",
 		name = "Cut",
-		type = PokemonTypes.NORMAL,
+		type = PokemonData.Types.NORMAL,
 		power = "50",
 		pp = "30",
 		accuracy = "95",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		iscontact = true,
 		summary = "Deals damage and has no secondary effect.",
 	},
 	{
 		id = "16",
 		name = "Gust",
-		type = PokemonTypes.FLYING,
+		type = PokemonData.Types.FLYING,
 		power = "40",
 		pp = "35",
 		accuracy = "100",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		summary = "Deals double damage if opponent is using Fly or Bounce.",
 	},
 	{
 		id = "17",
 		name = "Wing Attack",
-		type = PokemonTypes.FLYING,
+		type = PokemonData.Types.FLYING,
 		power = "60",
 		pp = "35",
 		accuracy = "100",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		iscontact = true,
 		summary = "Deals damage and has no secondary effect.",
 	},
 	{
 		id = "18",
 		name = "Whirlwind",
-		type = PokemonTypes.NORMAL,
-		power = NOPOWER,
+		type = PokemonData.Types.NORMAL,
+		power = Constants.NO_POWER,
 		pp = "20",
 		accuracy = "100",
-		category = MoveCategories.STATUS,
+		category = MoveData.Categories.STATUS,
 		priority = "-- 6",
 		summary = "Forces target to switch out with another random Pokemon. Fails against Suction Cups or Ingrain.",
 	},
 	{
 		id = "19",
 		name = "Fly",
-		type = PokemonTypes.FLYING,
+		type = PokemonData.Types.FLYING,
 		power = "70",
 		pp = "15",
 		accuracy = "95",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		iscontact = true,
 		summary = "Attacks on 2nd turn. Can still be hit by Gust, Sky Uppercut, Thunder, Twister, and Whirlwind.",
 	},
 	{
 		id = "20",
 		name = "Bind",
-		type = PokemonTypes.NORMAL,
+		type = PokemonData.Types.NORMAL,
 		power = "15",
 		pp = "20",
 		accuracy = "75",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		iscontact = true,
 		summary = "Deals damage and inflicts 1/16 of the target's max HP as damage for 2-5 turns.",
```

Comment:
Could maybe also mention that [binding moves](https://bulbapedia.bulbagarden.net/wiki/Category:Binding_moves) such as bind also trap the target, preventing running away / switching out while the move is in effect